<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="PlexedPiper">
<title>Suggested TMT processing pipeline. Variant 1.
 • PlexedPiper</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Suggested TMT processing pipeline. Variant 1.
">
<meta property="og:description" content="PlexedPiper">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">PlexedPiper</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.4.0</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/tmt_pipeline_for_PNNL_DMS_phos.html">Suggested TMT processing pipeline for PNNL DMS *phospho* data.
</a>
    <a class="dropdown-item" href="../articles/tmt_pipeline_for_PNNL_DMS_phos_MoTrPAC.html">Suggested TMT processing pipeline for PNNL DMS *phospho* data
for MoTrPAC project.
</a>
    <a class="dropdown-item" href="../articles/tmt_pipeline_for_PNNL_DMS_phos_pep_MoTrPAC.html">Suggested TMT processing pipeline for PNNL DMS *phospho* data
for MoTrPAC project. Peptide level.
</a>
    <a class="dropdown-item" href="../articles/tmt_pipeline_v1.html">Suggested TMT processing pipeline. Variant 1.
</a>
    <a class="dropdown-item" href="../articles/tmt_pipeline_v1_PNNL_DMS.html">Suggested TMT processing pipeline. Variant 1.
Version for PNNL DMS.
</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../news/index.html">Changelog</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/PNNL-Comp-Mass-Spec/PlexedPiper/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Suggested TMT processing pipeline. Variant 1.
</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/PNNL-Comp-Mass-Spec/PlexedPiper/blob/HEAD/vignettes/tmt_pipeline_v1.Rmd" class="external-link"><code>vignettes/tmt_pipeline_v1.Rmd</code></a></small>
      <div class="d-none name"><code>tmt_pipeline_v1.Rmd</code></div>
    </div>

    
    
<p><code>PlexedPiper</code> is the main library for the pipeline.
<code>MSnID</code> package is a flexible way of handling MS/MS
identifications. It handles MS/MS data filtering part.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://pnnl-comp-mass-spec.github.io/PlexedPiper/">PlexedPiper</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">MSnID</span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="data-source">Data source<a class="anchor" aria-label="anchor" href="#data-source"></a>
</h2>
<p>This pipeline shows how to process TMT data that is processed outside
of PNNL DMS. This implies that raw files processed by MSGF+ and MASIC
tools and the processing results are in corresponding directories. For
convienience the results of MSGF+ and MASIC processing are provided in a
companion <code>PlexedPiperTestData</code> package.</p>
</div>
<div class="section level2">
<h2 id="prepare-msms-ids">Prepare MS/MS IDs<a class="anchor" aria-label="anchor" href="#prepare-msms-ids"></a>
</h2>
<div class="section level3">
<h3 id="read-the-ms-gf-output">Read the MS-GF+ output<a class="anchor" aria-label="anchor" href="#read-the-ms-gf-output"></a>
</h3>
<p>This simply reads parsed to text output of MS-GF+ search engine. The
text files are collated together and the resulting
<code>data.frame</code> used to create MSnID object.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">path_to_MSGF_results</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span></span>
<span>   <span class="st">"extdata/global/msgf_output"</span>,</span>
<span>   package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span></span>
<span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/read_msgf_data.html">read_msgf_data</a></span><span class="op">(</span><span class="va">path_to_MSGF_results</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 1156754 at 31 % FDR</span></span>
<span><span class="co">## #peptides: 511617 at 61 % FDR</span></span>
<span><span class="co">## #accessions: 128378 at 98 % FDR</span></span></code></pre>
<p>This <code>MSnID</code> methods corrects for isotope selection error.
Occasionally, instrument selects not the lightest or monoisotopic peak,
but a peak with +1 or more C13. While MS-GF+ is still capable of
correctly identifying those, the downstream calculations of mass
measurement error need to be fixed. This method corrects mass
measurement errors accounting for selection of other than monoisotopic
peak.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MSnID/man/correct_peak_selection.html" class="external-link">correct_peak_selection</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="msms-id-filter-and-peptide-level">MS/MS ID filter and peptide level<a class="anchor" aria-label="anchor" href="#msms-id-filter-and-peptide-level"></a>
</h3>
<p>Main MS/MS filtering step. The
<code>filter_msgf_data_peptide_level</code> function wraps-up
<code><a href="https://rdrr.io/pkg/MSnID/man/optimize_filter.html" class="external-link">MSnID::optimize_filter</a></code> method and uses
<code>PepQValue</code> from MS-GF+ output and absolute deviation of mass
measurement error of parent ions (in ppm).</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 1156754 at 31 % FDR</span></span>
<span><span class="co">## #peptides: 511617 at 61 % FDR</span></span>
<span><span class="co">## #accessions: 128378 at 98 % FDR</span></span></code></pre>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/filter_msgf_data.html">filter_msgf_data_peptide_level</a></span><span class="op">(</span><span class="va">msnid</span>, <span class="fl">0.01</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 480204 at 0.45 % FDR</span></span>
<span><span class="co">## #peptides: 96637 at 1 % FDR</span></span>
<span><span class="co">## #accessions: 26926 at 8.7 % FDR</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="switching-annotation-from-refseq-to-gene-symbols">Switching annotation from RefSeq to gene symbols<a class="anchor" aria-label="anchor" href="#switching-annotation-from-refseq-to-gene-symbols"></a>
</h3>
<p>This RefSeq to gene symbols conversion step can be skipped if the
downstream analysis relies on RefSeq IDs.</p>
<p>RefSeq is a very redundant database and contains lots of tentative
splice isoforms and unconfirmed ORFs. As a result, a lot of peptides
(~60%) are shared across multiple RefSeq accessions. This adds to
uncertainity which exactly protein we are quantifying. One solution is
to switch to less redundant annotations like gene symbol. Moreover gene
symbols are more human-interpretable and quite commonly used in
downstream analysis.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/remap_accessions.html">remap_accessions_refseq_to_gene</a></span><span class="op">(</span><span class="va">msnid</span>, </span>
<span>                                         organism_name<span class="op">=</span><span class="st">"Rattus norvegicus"</span><span class="op">)</span></span></code></pre></div>
<p>This step converts sequences names in the fasta file used for MS/MS
searches from RefSeq to gene symbols. This is necessary if we are going
to count the number of peptide identifications per protein or protein
sequence coverage with accessions switched from RefSeq to gene
symbols.</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">path_to_FASTA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span></span>
<span>   <span class="st">"extdata/Rattus_norvegicus_NCBI_RefSeq_2018-04-10.fasta.gz"</span>,</span>
<span>   package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span></span>
<span><span class="va">temp_dir</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/tempfile.html" class="external-link">tempdir</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/files.html" class="external-link">file.copy</a></span><span class="op">(</span><span class="va">path_to_FASTA</span>, <span class="va">temp_dir</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] TRUE</span></span></code></pre>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">path_to_FASTA</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/file.path.html" class="external-link">file.path</a></span><span class="op">(</span><span class="va">temp_dir</span>, <span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/path.html" class="external-link">basename</a></span><span class="op">(</span><span class="va">path_to_FASTA</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">path_to_FASTA_gene</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/remap_accessions_fasta.html">remap_accessions_refseq_to_gene_fasta</a></span><span class="op">(</span></span>
<span>   <span class="va">path_to_FASTA</span>, organism_name<span class="op">=</span><span class="st">"Rattus norvegicus"</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="msms-id-filter-at-protein-level">MS/MS ID filter at protein level<a class="anchor" aria-label="anchor" href="#msms-id-filter-at-protein-level"></a>
</h3>
<p>A while ago proteomics field established hard-and-fast two peptide
per protein rule. That is if a protein identified with one peptide it is
not considered a confident identification. It works OK, but there is a
room for improvement. For example this rule really penalizes short
protens. Then there are some very long proteins (e.g. Titin 3.8 MDa)
that easily have more then two matching peptides even in reversed
sequence. Thus, here we propose to normalize number of peptides per
protein length and use that as a filtering criterion.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/compute_num_peptides_per_1000aa.html">compute_num_peptides_per_1000aa</a></span><span class="op">(</span><span class="va">msnid</span>, <span class="va">path_to_FASTA_gene</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 480204 at 0.45 % FDR</span></span>
<span><span class="co">## #peptides: 96637 at 1 % FDR</span></span>
<span><span class="co">## #accessions: 8140 at 7 % FDR</span></span></code></pre>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/filter_msgf_data.html">filter_msgf_data_protein_level</a></span><span class="op">(</span><span class="va">msnid</span>, <span class="fl">0.01</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 398445 at 0.2 % FDR</span></span>
<span><span class="co">## #peptides: 81353 at 0.35 % FDR</span></span>
<span><span class="co">## #accessions: 5369 at 1 % FDR</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="inference-of-parsimonious-protein-set">Inference of parsimonious protein set<a class="anchor" aria-label="anchor" href="#inference-of-parsimonious-protein-set"></a>
</h3>
<p>The situation when sequence of a certain peptide matches multiple
proteins adds complication to the downstream quantitative analysis as it
is not clear which protein product this peptide is originating from.
There are common ways for selecting a protein set. One is simply retain
only uniquely matching peptides (<code>unique_only=TRUE</code>). Shared
peptides are simply discarded in this case. Alternatively (in case of
<code>unique_only=FALSE</code>) the shared peptides assigned to the
proteins with larger number of uniquely mapping peptides. If there is a
choice between multiple proteins with equal number of uniquely mapping
peptides, the shared peptides is assigned to the first protein according
to alphanumeric order.</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MSnID/man/infer_parsimonious_accessions.html" class="external-link">infer_parsimonious_accessions</a></span><span class="op">(</span><span class="va">msnid</span>, unique_only<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/methods/show.html" class="external-link">show</a></span><span class="op">(</span><span class="va">msnid</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## MSnID object</span></span>
<span><span class="co">## Working directory: "."</span></span>
<span><span class="co">## #Spectrum Files:  48 </span></span>
<span><span class="co">## #PSMs: 302475 at 0.23 % FDR</span></span>
<span><span class="co">## #peptides: 73087 at 0.34 % FDR</span></span>
<span><span class="co">## #accessions: 4822 at 0.96 % FDR</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="remove-decoy-accessions">Remove decoy accessions<a class="anchor" aria-label="anchor" href="#remove-decoy-accessions"></a>
</h3>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">msnid</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/pkg/MSnID/man/apply_filter.html" class="external-link">apply_filter</a></span><span class="op">(</span><span class="va">msnid</span>, <span class="st">"!isDecoy"</span><span class="op">)</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="prepare-reporter-ion-intensities">Prepare reporter ion intensities<a class="anchor" aria-label="anchor" href="#prepare-reporter-ion-intensities"></a>
</h2>
<div class="section level3">
<h3 id="read-masic-ouput">Read MASIC ouput<a class="anchor" aria-label="anchor" href="#read-masic-ouput"></a>
</h3>
<p>MASIC is a tool for extracting ion intensities. With proper parameter
settings it can be used for extracting TMT (or iTRAQ) reporter ion
intensities. In addition it reporter a number of other helpful metrics.
Of the most interest are interference score of the parent ion level and
S/N at reporter ion level. Interference score reflects what proportion
of ion population isolated for fragmentation is due to the targeted ion.
The other words 1 - InterferenceScore is due to co-isolated species that
have similar elution time and parent ion m/z. S/N at reporter level is
the metric that computed by Thermo software.</p>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">path_to_MASIC_results</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata/global/masic_output"</span>, package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span></span>
<span><span class="va">masic_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/read_masic_data.html">read_masic_data</a></span><span class="op">(</span><span class="va">path_to_MASIC_results</span>, interference_score<span class="op">=</span><span class="cn">TRUE</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">masic_data</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                             Dataset ScanNumber Collision.Mode</span></span>
<span><span class="co">## 1 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          2            hcd</span></span>
<span><span class="co">## 2 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          3            hcd</span></span>
<span><span class="co">## 3 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          4            hcd</span></span>
<span><span class="co">## 4 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          5            hcd</span></span>
<span><span class="co">## 5 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          6            hcd</span></span>
<span><span class="co">## 6 MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02          8            hcd</span></span>
<span><span class="co">##   ParentIonMZ BasePeakIntensity BasePeakMZ ReporterIonIntensityMax Ion_126.128</span></span>
<span><span class="co">## 1      407.74         227695.44   407.7408                92236.87    70562.39</span></span>
<span><span class="co">## 2      396.72          59127.97   529.2943                34294.90    23706.89</span></span>
<span><span class="co">## 3      438.23         110444.82   362.2239                14053.40    12459.86</span></span>
<span><span class="co">## 4      481.50          37082.72   206.4663                    0.00        0.00</span></span>
<span><span class="co">## 5      549.28          21077.05   128.1285                21077.05        0.00</span></span>
<span><span class="co">## 6      388.72          40605.85   356.7193                 8087.76     6166.82</span></span>
<span><span class="co">##   Ion_127.125 Ion_127.131 Ion_128.128 Ion_128.134 Ion_129.131 Ion_129.138</span></span>
<span><span class="co">## 1    24864.62    17165.80    35625.00    92236.87     9640.23     8578.05</span></span>
<span><span class="co">## 2    13559.32     5856.83    16322.71    34294.90     4853.11     7938.24</span></span>
<span><span class="co">## 3    11785.91    10932.51    10653.32    12328.62     5959.86     9905.82</span></span>
<span><span class="co">## 4        0.00        0.00        0.00        0.00        0.00        0.00</span></span>
<span><span class="co">## 5    10998.67        0.00    21077.05     2725.50        0.00        0.00</span></span>
<span><span class="co">## 6     1371.27     2418.35     8087.76     5485.35        0.00        0.00</span></span>
<span><span class="co">##   Ion_130.135 Ion_130.141 Ion_131.138 Weighted.Avg.Pct.Intensity.Correction</span></span>
<span><span class="co">## 1     6996.69    11833.07    32281.34                                     0</span></span>
<span><span class="co">## 2        0.00     1465.03    18182.27                                     0</span></span>
<span><span class="co">## 3     8387.04    11166.70    14053.40                                     0</span></span>
<span><span class="co">## 4        0.00        0.00        0.00                                     0</span></span>
<span><span class="co">## 5        0.00        0.00     6800.70                                     0</span></span>
<span><span class="co">## 6     1543.48     1943.96     7436.60                                     0</span></span>
<span><span class="co">##   Ion_126.128_SignalToNoise Ion_127.125_SignalToNoise Ion_127.131_SignalToNoise</span></span>
<span><span class="co">## 1                     71.47                     25.17                     17.38</span></span>
<span><span class="co">## 2                     26.12                     14.94                      6.45</span></span>
<span><span class="co">## 3                     12.40                     11.75                     10.90</span></span>
<span><span class="co">## 4                        NA                        NA                        NA</span></span>
<span><span class="co">## 5                        NA                      9.19                        NA</span></span>
<span><span class="co">## 6                      6.92                      1.54                      2.71</span></span>
<span><span class="co">##   Ion_128.128_SignalToNoise Ion_128.134_SignalToNoise Ion_129.131_SignalToNoise</span></span>
<span><span class="co">## 1                     36.04                     93.32                      9.75</span></span>
<span><span class="co">## 2                     17.97                     37.77                      5.34</span></span>
<span><span class="co">## 3                     10.64                     12.31                      5.96</span></span>
<span><span class="co">## 4                        NA                        NA                        NA</span></span>
<span><span class="co">## 5                     17.57                      2.27                        NA</span></span>
<span><span class="co">## 6                      9.04                      6.13                        NA</span></span>
<span><span class="co">##   Ion_129.138_SignalToNoise Ion_130.135_SignalToNoise Ion_130.141_SignalToNoise</span></span>
<span><span class="co">## 1                      8.67                      7.07                     11.96</span></span>
<span><span class="co">## 2                      8.74                        NA                      1.61</span></span>
<span><span class="co">## 3                      9.91                      8.40                     11.18</span></span>
<span><span class="co">## 4                        NA                        NA                        NA</span></span>
<span><span class="co">## 5                        NA                        NA                        NA</span></span>
<span><span class="co">## 6                        NA                      1.72                      2.16</span></span>
<span><span class="co">##   Ion_131.138_SignalToNoise Ion_126.128_Resolution Ion_127.125_Resolution</span></span>
<span><span class="co">## 1                     32.71                  44102                  42700</span></span>
<span><span class="co">## 2                     19.93                  42702                  41100</span></span>
<span><span class="co">## 3                     14.13                  42006                  40702</span></span>
<span><span class="co">## 4                        NA                     NA                     NA</span></span>
<span><span class="co">## 5                      5.66                     NA                  40302</span></span>
<span><span class="co">## 6                      8.26                  40000                  26400</span></span>
<span><span class="co">##   Ion_127.131_Resolution Ion_128.128_Resolution Ion_128.134_Resolution</span></span>
<span><span class="co">## 1                  42100                  41800                  44404</span></span>
<span><span class="co">## 2                  37000                  40400                  43404</span></span>
<span><span class="co">## 3                  41402                  40700                  40400</span></span>
<span><span class="co">## 4                     NA                     NA                     NA</span></span>
<span><span class="co">## 5                     NA                  42102                  46600</span></span>
<span><span class="co">## 6                  30400                  40400                  44300</span></span>
<span><span class="co">##   Ion_129.131_Resolution Ion_129.138_Resolution Ion_130.135_Resolution</span></span>
<span><span class="co">## 1                  40500                  39500                  36800</span></span>
<span><span class="co">## 2                  36400                  39700                     NA</span></span>
<span><span class="co">## 3                  38800                  40200                  38900</span></span>
<span><span class="co">## 4                     NA                     NA                     NA</span></span>
<span><span class="co">## 5                     NA                     NA                     NA</span></span>
<span><span class="co">## 6                     NA                     NA                  28800</span></span>
<span><span class="co">##   Ion_130.141_Resolution Ion_131.138_Resolution ParentIonIndex       MZ</span></span>
<span><span class="co">## 1                  41100                  42302              0 407.7424</span></span>
<span><span class="co">## 2                  29800                  41802              1 396.7176</span></span>
<span><span class="co">## 3                  40400                  41002              2 438.2270</span></span>
<span><span class="co">## 4                     NA                     NA              3 481.5049</span></span>
<span><span class="co">## 5                     NA                  40300              4 549.2793</span></span>
<span><span class="co">## 6                  28500                  38700              5 388.7199</span></span>
<span><span class="co">##   SurveyScanNumber OptimalPeakApexScanNumber PeakApexOverrideParentIonIndex</span></span>
<span><span class="co">## 1                1                        12                             -1</span></span>
<span><span class="co">## 2                1                        12                             -1</span></span>
<span><span class="co">## 3                1                       131                             18</span></span>
<span><span class="co">## 4                1                        23                             -1</span></span>
<span><span class="co">## 5                1                        15                             -1</span></span>
<span><span class="co">## 6                7                        12                             -1</span></span>
<span><span class="co">##   CustomSICPeak PeakScanStart PeakScanEnd PeakScanMaxIntensity PeakMaxIntensity</span></span>
<span><span class="co">## 1             0             1          19                   12          2901600</span></span>
<span><span class="co">## 2             0             1          19                   12          2181900</span></span>
<span><span class="co">## 3             0             1         114                  107          8255600</span></span>
<span><span class="co">## 4             0             1          52                   23           401824</span></span>
<span><span class="co">## 5             0             1          16                   15           363656</span></span>
<span><span class="co">## 6             0             1          30                   12           478135</span></span>
<span><span class="co">##   PeakSignalToNoiseRatio FWHMInScans  PeakArea ParentIonIntensity</span></span>
<span><span class="co">## 1               211.0000           5  50422000            2579600</span></span>
<span><span class="co">## 2                19.6900           5  34508000            1690600</span></span>
<span><span class="co">## 3                 9.4650          44 444610000             658727</span></span>
<span><span class="co">## 4                27.9900          23  16244000             344491</span></span>
<span><span class="co">## 5                 0.6998           5   5941200             347071</span></span>
<span><span class="co">## 6                28.7100          19  10718000             291189</span></span>
<span><span class="co">##   PeakBaselineNoiseLevel PeakBaselineNoiseStDev PeakBaselinePointsUsed</span></span>
<span><span class="co">## 1                  13750                  97562                  10113</span></span>
<span><span class="co">## 2                 110841                1120000                  10166</span></span>
<span><span class="co">## 3                 872195                2620000                  10129</span></span>
<span><span class="co">## 4                  14356                  65777                  10109</span></span>
<span><span class="co">## 5                 519640                1990000                  10109</span></span>
<span><span class="co">## 6                  16653                 142562                  10142</span></span>
<span><span class="co">##   StatMomentsArea CenterOfMassScan PeakStDev PeakSkew PeakKSStat</span></span>
<span><span class="co">## 1        47031000               11      5.68 -0.18920     0.3528</span></span>
<span><span class="co">## 2        31578000               11      5.59 -0.21720     0.3469</span></span>
<span><span class="co">## 3       343470000               82     22.80 -0.62610     1.0990</span></span>
<span><span class="co">## 4        14899000               26     14.10  0.05387     0.5036</span></span>
<span><span class="co">## 5         2528900               15      2.49 -0.58330     0.4302</span></span>
<span><span class="co">## 6         9961100               16      8.60 -0.05055     0.2833</span></span>
<span><span class="co">##   StatMomentsDataCountUsed InterferenceScore</span></span>
<span><span class="co">## 1                       10            0.9961</span></span>
<span><span class="co">## 2                       10            0.9925</span></span>
<span><span class="co">## 3                       89            1.0000</span></span>
<span><span class="co">## 4                       39            1.0000</span></span>
<span><span class="co">## 5                        5            1.0000</span></span>
<span><span class="co">## 6                       18            0.9688</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="filtering-masic-data">Filtering MASIC data<a class="anchor" aria-label="anchor" href="#filtering-masic-data"></a>
</h3>
<p>In this case (currently recommended filters) we require that at least
50% of ion population is due to the targeted ion and no filter at S/N
level.</p>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">masic_data</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 1598123</span></span></code></pre>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">masic_data</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/filter_masic_data.html">filter_masic_data</a></span><span class="op">(</span><span class="va">masic_data</span>, <span class="fl">0.5</span>, <span class="fl">0</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/pkg/BiocGenerics/man/nrow.html" class="external-link">nrow</a></span><span class="op">(</span><span class="va">masic_data</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 1523732</span></span></code></pre>
</div>
</div>
<div class="section level2">
<h2 id="creating-cross-tab">Creating cross-tab<a class="anchor" aria-label="anchor" href="#creating-cross-tab"></a>
</h2>
<div class="section level3">
<h3 id="laying-out-study-desing">Laying out study desing<a class="anchor" aria-label="anchor" href="#laying-out-study-desing"></a>
</h3>
<p>To convert from PSMs and reporter ion intensities to meaningful
quantitative data it is necessary to know what are the samples in the
reporter channels and what is the intended reference channel (or
combination of channels). The entire study desing captured by three
tables - fractions, samples, references.</p>
</div>
<div class="section level3">
<h3 id="fractions">Fractions<a class="anchor" aria-label="anchor" href="#fractions"></a>
</h3>
<p>This links dataset to original plex.</p>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://readr.tidyverse.org" class="external-link">readr</a></span><span class="op">)</span></span>
<span><span class="va">fractions</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://readr.tidyverse.org/reference/read_delim.html" class="external-link">read_tsv</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata/study_design/fractions.txt"</span>, package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">fractions</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 6 × 2</span></span></span>
<span><span class="co">##   Dataset                                           PlexID</span></span>
<span><span class="co">##   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                                             <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> MoTrPAC_Pilot_TMT_W_S1_01_12Oct17_Elm_AQ-17-09-02 S1    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">2</span> MoTrPAC_Pilot_TMT_W_S1_02_12Oct17_Elm_AQ-17-09-02 S1    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">3</span> MoTrPAC_Pilot_TMT_W_S1_05_12Oct17_Elm_AQ-17-09-02 S1    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">4</span> MoTrPAC_Pilot_TMT_W_S1_03_12Oct17_Elm_AQ-17-09-02 S1    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">5</span> MoTrPAC_Pilot_TMT_W_S1_04_12Oct17_Elm_AQ-17-09-02 S1    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">6</span> MoTrPAC_Pilot_TMT_W_S1_06_12Oct17_Elm_AQ-17-09-02 S1</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="samples">Samples<a class="anchor" aria-label="anchor" href="#samples"></a>
</h3>
<p>This links plex ID, reporter channel with samples. There is a minor
difference between ReporterAlias and MeasurementName. MeasurementName is
the sample name that will be finally reported. ReporterAlias is an
itermediate between ReporterName and MeasurementName and is used for
defining what the reference is.</p>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">samples</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://readr.tidyverse.org/reference/read_delim.html" class="external-link">read_tsv</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata/study_design/samples.txt"</span>, package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">samples</span>,<span class="fl">10</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 10 × 5</span></span></span>
<span><span class="co">##    PlexID QuantBlock ReporterName ReporterAlias MeasurementName</span></span>
<span><span class="co">##    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 1</span> S1              1 126          R_01          R_01           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 2</span> S1              1 127N         R_03          R_03           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 3</span> S1              1 127C         R_02          R_02           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 4</span> S1              1 128N         R_05          R_05           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 5</span> S1              1 128C         R_04          R_04           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 6</span> S1              1 129N         S_02          S_02           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 7</span> S1              1 129C         S_01          S_01           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 8</span> S1              1 130N         S_04          S_04           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;"> 9</span> S1              1 130C         S_03          S_03           </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">10</span> S1              1 131          ref           <span style="color: #BB0000;">NA</span></span></span></code></pre>
</div>
<div class="section level3">
<h3 id="references">References<a class="anchor" aria-label="anchor" href="#references"></a>
</h3>
<p>Reference can be a certain channel, average of multiple channels or
1. In general form it is an expression with ReporterAlias names as
variables. It is evaluated for each PlexID/QuantBlock combination and
applied to divide reporter ion intensities within corresponding
PlexID/QuantBlock. QuantBlock can be though of as a way of defining
sub-plex. In a typical TMT experiment QuantBlock is always 1. In case of
5 pairwise comparisons within TMT10, there will be 5 QuantBlocks (1..5)
with reference for each QuantBlock.</p>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">references</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://readr.tidyverse.org/reference/read_delim.html" class="external-link">read_tsv</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata/study_design/references.txt"</span>, package <span class="op">=</span> <span class="st">"PlexedPiperTestData"</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">references</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## <span style="color: #949494;"># A tibble: 2 × 3</span></span></span>
<span><span class="co">##   PlexID QuantBlock Reference</span></span>
<span><span class="co">##   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">1</span> S1              1 ref      </span></span>
<span><span class="co">## <span style="color: #BCBCBC;">2</span> S2              1 ref</span></span></code></pre>
</div>
<div class="section level3">
<h3 id="creating-quantitative-cross-tab">Creating quantitative cross-tab<a class="anchor" aria-label="anchor" href="#creating-quantitative-cross-tab"></a>
</h3>
<p>Final step when MS/MS IDs and reporter ions linked together,
aggregated down to peptide or accession (i.e. protein) level. To retain
protein IDs while aggregating to peptide level we suggest
<code>aggregation_level &lt;- c("accession","peptide")</code></p>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">aggregation_level</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"accession"</span><span class="op">)</span></span>
<span><span class="va">quant_cross_tab</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/create_crosstab.html">create_crosstab</a></span><span class="op">(</span><span class="va">msnid</span>, </span>
<span>                                   <span class="va">masic_data</span>, </span>
<span>                                   <span class="va">aggregation_level</span>, </span>
<span>                                   <span class="va">fractions</span>, <span class="va">samples</span>, <span class="va">references</span><span class="op">)</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/dim.html" class="external-link">dim</a></span><span class="op">(</span><span class="va">quant_cross_tab</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] 4764   18</span></span></code></pre>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">quant_cross_tab</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##              R_01        R_02       R_03         R_04       R_05       R_06</span></span>
<span><span class="co">## ATP8  -0.19479833 -0.40351693 -0.7899082 -0.207686309 -0.5735321 -0.6217460</span></span>
<span><span class="co">## Aagab -0.33114648 -0.09698122 -0.4610205 -0.204455063 -0.3674259 -0.9622379</span></span>
<span><span class="co">## Aak1  -0.12524883 -0.04935325 -0.3961333  0.006220386 -0.1831229 -0.8278655</span></span>
<span><span class="co">## Aamdc -0.18262236 -0.14712884 -0.4295235 -0.338216384 -0.5193325 -0.7662433</span></span>
<span><span class="co">## Aamp   0.01319057 -0.05586270 -0.4874643 -0.176035341 -0.2877920 -0.9128142</span></span>
<span><span class="co">## Aar2  -0.28239827 -0.05731137 -0.4382888 -0.312729380 -0.5601654 -0.7180116</span></span>
<span><span class="co">##             R_07       R_08         R_09        S_01       S_02       S_03</span></span>
<span><span class="co">## ATP8  -0.6862703 -0.3539212 -0.239588729 -1.23906148 -1.0119484 -0.7436739</span></span>
<span><span class="co">## Aagab -0.8219094 -0.6039567 -0.349913889 -0.16655998 -1.0566135 -0.6460940</span></span>
<span><span class="co">## Aak1  -0.5477375 -0.4730509  0.032356743 -0.02601848 -0.7284031 -0.5183336</span></span>
<span><span class="co">## Aamdc -0.6260984 -0.4093530 -0.004382412 -0.18326078 -0.8802873 -0.5951816</span></span>
<span><span class="co">## Aamp  -0.7298059 -0.5725573 -0.127739120  0.03925511 -0.5639567 -0.5788605</span></span>
<span><span class="co">## Aar2  -0.6492230 -0.3802387 -0.033934887 -0.27326401 -0.8040756 -0.5338878</span></span>
<span><span class="co">##             S_04        S_05       S_06       S_07       S_08       S_09</span></span>
<span><span class="co">## ATP8  -0.8203905  0.18483797 -0.1652751 -0.3545360 -0.6421465 -0.4344390</span></span>
<span><span class="co">## Aagab -0.9723428 -0.72798837 -0.7156128 -0.1366306 -1.0352613 -0.6912676</span></span>
<span><span class="co">## Aak1  -1.0432754 -0.19864328 -0.3533702 -0.3482648 -0.5589898 -0.5497933</span></span>
<span><span class="co">## Aamdc -0.9138052 -0.19684213 -0.2303194 -0.2628076 -0.7913746 -0.5035342</span></span>
<span><span class="co">## Aamp  -0.9582277 -0.28850123 -0.3668970 -0.5391545 -0.8578482 -0.5955917</span></span>
<span><span class="co">## Aar2  -0.9288846 -0.01109715 -0.3226456 -0.2250253 -0.7562436 -0.4057189</span></span></code></pre>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by Vladislav Petyuk.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
